test/fixtures/linter/rules/XMLInJs/SapUIView.js

```diff
@@ -0,0 +1,38 @@
+sap.ui.define([], () => {
+	const oView = sap.ui.view({
+		type: "XML",
+		definition: `<mvc:View 
+			controllerName="ui5app.controller.Home" 
+			displayBlock="true"
+			xmlns="sap.m"
+			xmlns:mvc="sap.ui.core.mvc"
+			xmlns:core="sap.ui.core"> 
+				<Button tap=".sayHello">
+			</mvc:View>`,
+	});
+	oView.placeAt("content");
+	
+	const oView2 = sap.ui.view({
+		type: "XML",
+		viewContent: `<mvc:View 
```

Comment:
Please also include a test case where a normal string literal is used within one line. I could reproduce a wrong column information when using this:
`viewContent: '<mvc:View controllerName="ui5app.controller.Home" displayBlock="true" xmlns="sap.m" xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core"><Button tap=".sayHello"></mvc:View>'`

Reply:
Done

Reply:
To me the column info seems wrong. It is 146 but should be 162, isn't it?

Reply:
Thanks! Now it's fixed